node/src/node/stats.cpp

```diff
@@ -10,27 +10,51 @@
 
 namespace cocaine {
 
+namespace {
+
+const char name_requests_accepted[] = "{}.requests.accepted";
+const char name_requests_rejected[] = "{}.requests.rejected";
+const char name_slaves_spawned[] = "{}.slaves.spawned";
+const char name_slaves_crashed[] = "{}.slaves.crashed";
+const char name_rate[] = "{}.rate";
+const char name_queue_depth_average[] = "{}.queue.depth_average";
+const char name_timings[] = "{}.timings";
+
+} // namespace
+
 stats_t::stats_t(context_t& context, const std::string& name, std::chrono::high_resolution_clock::duration interval):
+    name{name},
+    metrics_hub(context.metrics_hub()),
     requests{
-        context.metrics_hub().counter<std::int64_t>(cocaine::format("{}.requests.accepted", name)),
-        context.metrics_hub().counter<std::int64_t>(cocaine::format("{}.requests.rejected", name))
+        metrics_hub.counter<std::int64_t>(cocaine::format(name_requests_accepted, name)),
+        metrics_hub.counter<std::int64_t>(cocaine::format(name_requests_rejected, name))
     },
     slaves{
-        context.metrics_hub().counter<std::int64_t>(cocaine::format("{}.slaves.spawned", name)),
-        context.metrics_hub().counter<std::int64_t>(cocaine::format("{}.slaves.crashed", name))
+        metrics_hub.counter<std::int64_t>(cocaine::format(name_slaves_spawned, name)),
+        metrics_hub.counter<std::int64_t>(cocaine::format(name_slaves_crashed, name))
     },
-    meter(context.metrics_hub().meter(cocaine::format("{}.rate", name))),
+    meter(metrics_hub.meter(cocaine::format(name_rate, name))),
     queue_depth(std::make_shared<metrics::usts::ewma_t>(interval)),
-    queue_depth_gauge(context.metrics_hub()
+    queue_depth_gauge(metrics_hub
         .register_gauge<double>(
-            cocaine::format("{}.queue.depth_average", name),
+            cocaine::format(name_queue_depth_average, name),
             {},
             std::bind(&metrics::usts::ewma_t::get, queue_depth)
         )
     ),
-    timer(context.metrics_hub().timer<metrics::accumulator::decaying::exponentially_t>(cocaine::format("{}.timings", name)))
+    timer(metrics_hub.timer<metrics::accumulator::decaying::exponentially_t>(cocaine::format(name_timings, name)))
 {
     queue_depth->add(0);
 }
 
-}  // namespace cocaine
+auto stats_t::deregister() -> void {
+    metrics_hub.remove<std::atomic<std::int64_t>>(cocaine::format(name_requests_accepted, name), {});
```

Comment:
Not an issue at all, but what will happen in case of template parameter misspelling, eg:  `std::int64_t ->  std::uint_64_t` ? Parameters used in few places and it is possibility to make in error in naming, may be predefine an aliases for them?